src/InformaticsGateway/Program.cs

```diff
@@ -175,5 +185,29 @@ internal static void InitializeDatabase(IHost host)
                     webBuilder.UseStartup<Startup>();
                 })
                 .UseNLog();
+
+
+        private static NLog.Logger ConfigureNLog()
+        {
+            var version = typeof(Program).Assembly;
+            var assemblyVersionNumber = version.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion ?? "0.0.1";
+
+            LayoutRenderer.Register("servicename", logEvent => Assembly.GetEntryAssembly()?.GetName().Name);
+            LayoutRenderer.Register("serviceversion", logEvent => assemblyVersionNumber);
+            LayoutRenderer.Register("machinename", logEvent => Environment.MachineName);
+
+            var nLog = LogManager.Setup().LoadConfigurationFromAppSettings().GetCurrentClassLogger();
+            var target = ((NLog.Targets.Wrappers.AsyncTargetWrapper)nLog.Factory.Configuration

```

Comment:
Can line 200-207 be configured in nlog.config?

Reply:
good point, moved now.